packages/react/src/patterns/sections/CardSection/README.md

```diff
@@ -41,6 +41,7 @@ ReactDOM.render(<App />, document.querySelector('#app'));
 | `heading` | YES      | String    | null          | Cards with or without images heading.                                                                                                                                |
 | `theme`   | NO       | String    | 'white'       | Color theme for pattern. See `theme` below.                                                                                                                          |
 | `cards`   | YES      | Array     | null          | Array of card objects. See [Card](https://github.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/react/src/patterns/sub-patterns/Card/README.md) 👀 |
+| `cta`     | NO       | Object    | null          | Cta options. See [Card] README with above link.                                                                                                                      |
```

Comment:
Is `[card]` intentionally in brackets`? CTA should be all caps because it's an acronym

Reply:
my bad done.